kateryna-tolochko/README.md

```diff
@@ -0,0 +1,91 @@
+
+![Filters](https://i.ibb.co/C2BHM3P/screen5.png)
+
+## Step 4. Select a product
+
+1. When you find a product that interests you, click it to view the product details page.
+
+2. Read the product description, specification, and reviews from previous customers.
+
+3. If the product has different variations (e.g., size, color), make sure to select the desired options before proceeding.
+
+## Step 5. Place an order
+
+1. On the product details page, click **Buy Now**.
+
+2. Provide shipping address.
+
+3. Select a method for payment. Enter the payment information, card info. Click **Confirm**.
+
+4. Click **Place Order**.
+
+## Step 6. Track your order
+
+After your payment is confirmed, the seller will prepare your order for shipment. Once the seller ships your order, you will be able to track the progress of your order.
+
+1. Hover over your account on the top right and go to **My Orders**.
+
+   !['Account' menu](https://i.ibb.co/Yj7Tbj2/screen3.png)
+
+2. Locate the item that you want to track and then click **Track order**.
+
+3. You will now be able to see the current status of your order and the AliExpress tracking number.
+
+## Step 7. Confirm receipt
+
+Once you receive your order and are satisfied with the products, confirm the receipt on AliExpress.
+
+1. Go to **My Orders**.
+
+2. Locate the order you would like to confirm as received and click **Confirm receipt**. This button is only available until the buyer protection term expires.
+
+3. A new window will appear asking if you want to confirm delivery. Note that this step is not reversible, as it closes and completes the shipment.
+
+4. If you are ready to confirm delivery and have physically received your order, click **Confirm** to complete this step.
+
+## Step 8. Resolve issues
+
+If you received damaged items, incorrect sizing, or experienced non-delivery, you can open a dispute until the buyer protection term expires.
+
+1. To open a dispute, go to **My Orders**, locate the relevant order, click **Order details**.
+
+2. Click **Refund/Return**.
+
+3. Follow the instructions provided to communicate with the seller and find a resolution.
+
+## Support and assistance resources
+
+If you have any questions or need assistance, here are some helpful links:
+
+- AliExpress Help Center: You can visit the [AliExpress Help Center](https://customerservice.aliexpress.com/home?language=en&from=byr_common&hcMapRule=aeMapRule) for answers to frequently asked questions, guides, and tutorials on using the platform.
+
+- Customer Service Chat: If you need personalized assistance or have specific issues related to your account or orders, you can contact an AliExpress agent. Go to [AliExpress Help Center](https://customerservice.aliexpress.com/home?language=en&from=byr_common&hcMapRule=aeMapRule), scroll down to the bottom of the page and click **Online Service**.
```

Comment:
Great job in general, Kateryna! Good knowledge of English and quite structured content. If possible, always amend your description with corresponding screenshots since bare text can be boring for reading and insufficient for understanding the whole picture you're talking about. Good luck!